READMEs/store-ipfs-huggingface-flow.md

```diff
@@ -0,0 +1,113 @@
+<!--
+Copyright 2022 Ocean Protocol Foundation
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Quickstart: Store and Discover Flow
+
+This quickstart describes how to encrypt an asset, store on IPFS and upload to HuggingFace for discoverability. 
+
+Here are the steps:
+
+1. Setup
+2. Alice encrypts a dataset and uploads to Filecoin and HuggingFace
+3. Alice publishes an asset
+4. Bob downloads datasets
+
+Let's go through each step.
+
+## 1. Setup
+
+### First steps
+
+To get started with this guide, please refer to [data-nfts-and-datatokens-flow](data-nfts-and-datatokens-flow.md) and complete the following steps :
+- [x] Setup : Prerequisites
+- [x] Setup : Download barge and run services
+- [x] Setup : Install the library from v4 sources
+
+### Set envvars
+
+We also need to set two additional envvars for using decentalized storage and HuggingFace. To obtain API keys, you will need to create accounts at [web3.storage](https://web3.storage/) or [Estuary](https://docs.estuary.tech/get-invite-key), and [HuggingFace](https://huggingface.co). 
+
+```console
+# Set envvars
+export STORAGE_KEY=<ADD WEB3.STORAGE OR ESTUARY.TECH KEY HERE>
+export HF_KEY=<ADD HUGGINGFACE KEY HERE>
+
+```
+
+## 2. Alice encrypts a dataset and uploads to Filecoin and HuggingFace
+
+Then in the same python console:
+```python
+
+object_path = "ocean_lib/storage_provider/test/hello.txt"
+with open(object_path, "rb") as f:
+    files = f.read()
+
+# Encrypt file(s) using provider
+from ocean_lib.data_provider.data_encryptor import DataEncryptor
+encrypt_response = DataEncryptor.encrypt(
+    files,
+    ocean.config.provider_url,
+)
+encrypted_files = encrypt_response.content.decode("utf-8")
+
+# Upload encrypted file to decentralized storage 
+from ocean_lib.storage_provider.storage_provider import StorageProvider
+storage_provider = StorageProvider()
+storage_response = storage_provider.upload(encrypted_files.encode("utf-8"))
+cid = storage_response.json()['cid']
+
+from ocean_lib.structures.file_objects import IpfsFile
+ipfs_file = [IpfsFile(hash=cid)]
+
+# Upload encrypted file to HuggingFace for better discoverability
+from ocean_lib.storage_provider.huggingface_hub import HuggingFaceHub
+huggingface = HuggingFaceHub()
+repo_url = huggingface.upload(encrypted_files.encode("utf-8"), object_name='hello', object_type='dataset')
+```
+
+## 3. Alice publishes an asset
+
+```python
+# Specify metadata and services, using the Branin test dataset
+metadata = {
```

Comment:
oh, so this PR is pretty old. We have added a lot of helpers to avoid these many imports and statements. Please look at the other readmes for the one-liner and use that instead.